### ISSUE TITLE
Is a typo in documentation?

### ISSUE THREAD

**Issue by AliM1988** [freqtrade/freqtrade#9916]
In https://www.freqtrade.io/en/2019.8/data-analysis/#change-directory-to-root I think there is a typo.

```
import os
from pathlib import Path

# Change directory
# Modify this cell to insure that the output shows the correct path.
# Define all paths relative to the project root shown in the cell output
project_root = "somedir/freqtrade"
i=0
try:
    os.chdirdir(project_root)
    assert Path('LICENSE').is_file()
except:
    while i<4 and (not Path('LICENSE').is_file()):
        os.chdir(Path(Path.cwd(), '../'))
        i+=1
    project_root = Path.cwd()
print(Path.cwd())
```
There's a typo in the line (os.chdirdir should be os.chdir). It's likely meant to change the directory using os.chdir()

**Comment by xmatthias** [comment]
Please get used to filling out the issue template.  Those who frequently omit the template may have their issues closed without comment.
We're asking some very basic questions - by deleting the issue template, you're explicitly choosing not to answer our questions.

Filling out the template helps reduce our workload, and we greatly appreciate that.

---


Your link goes to a documentation page for a version that's 4 (!) years old.

if you go to [the same, current page](https://www.freqtrade.io/en/stable/data-analysis/#change-directory-to-root) - then it's been fixed (most likely - 3.5 years ago - but i didn't actually check for this).

You can switch the version in the documentation in the bottom right corner - and I'd advise to use a documentation aligned with your version - which i hope isn't 4 years old - as then connections to the exchange might be problematic ...


![image](https://github.com/freqtrade/freqtrade/assets/5024695/17a421b7-a131-45d8-8c6f-4d0351ed1e6b)

**Comment by AliM1988** [comment]
Thank you. You are right about documentation. But I saw this typo in (user_data/notebooks/strategy_analysis_example.ipynb) in docker version. Actually I saw this typo there, at first.

Also, the typo there is in following link:

https://github.com/freqtrade/freqtrade/blob/develop/docs/strategy_analysis_example.md
